Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -79,10 +79,17 @@ int toInt(const std::string_view input, bool acceptSpaces) {
     while (*txt == ' ') {
       ++txt;
       --sz;
+      // have we run off the end of the view?
+      if (sz < 1U) {
+        // If we have a blank input string, should this throw or return 0?
+        // throw boost::bad_lexical_cast();
```

Comment:
I believe that the docs say that empty strings are zero, so what you have is right.
```suggestion
```